nav2_experimental/nav2_rl/nav2_turtlebot3_rl/navigation_task_env.py

```diff
@@ -86,20 +131,26 @@ def set_random_robot_pose(self):
         self.set_entity_state_pose('turtlebot3_waffle', self.get_random_pose())
 
     def set_random_goal_pose(self):
+        '''
+        Generates a random goal pose, and sets this goal pose in the Gazebo env.
+        '''
         self.goal_pose = self.get_random_pose()
         self.set_entity_state_pose('goal_pose', self.goal_pose)
 
     def get_random_pose(self):
         random_pose = Pose()
         yaw = random.uniform(0, pi * 2)
 
-        random_pose.position.x = random.uniform(-3, 3)
-        random_pose.position.y = random.uniform(-3, 3)
+        random_pose.position.x = random.uniform(-2, 2)
```

Comment:
This is hard-coding assumptions about the map. @mhpanah Do you have a plan for how to make this usable with other maps?

Reply:
@crdelsey Yes, this is only good for this map.  We plan to replace this with the collision checker function.   Basically we generate a random pose and then will check to see if it is colliding with any obstacles. If it is we will generate a new pose.